src/components/feed/reducer.js

```diff
@@ -61,6 +62,20 @@ export default (state = INITIAL_STATE, action) => {
 				],
 			};
 		}
+		case SET_COMMENT_POST: {
+			return {
+				...state,
+				allPosts: state.allPosts.map((post, i) =>
+					post.id === action.payload.id
+						? {
+								...post,
+								commentCount:
+									state.allPosts[i].commentCount + 1,
+						  }
+						: post,
+				),
```

Comment:
wrapper this logic in a function, its a bit confusing the way that it is laid out